rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -108,6 +109,15 @@ components:
       required: true
       schema:
         $ref: '#/components/schemas/Version'
+    Last-Modified:
+      name: Last-Modified
+      in: header
+      description: >
+        The RFC 3339, section 5.6 date-time format string of the last
+        modified time of the file(s) to be ingested.
+      required: true
+      schema:
+        $ref: '#/components/schemas/Date-Time'
```

Comment:
✏️ I don't think it's necessary to extract this schema because it's a defined format in OpenAPI.

```yml
Last-Modified:
  name: Last-Modified
  in: header
  description: >
    The last modified time of the product to be ingested.
  required: true
  schema:
    type: string
    format: date-time
    example: 2019-05-09T14:47:13.101+0000
```